ui/src/layout/Scans/ScanConfigWizardModal/CustomQueryBuilder/CustomQueryBuilder.scss

```diff
@@ -0,0 +1,229 @@
+  box-shadow: 0 0 0 2px rgba(5, 145, 255, 0.1);
+  outline: 0;
+}
+
+.ant-select:not(.ant-select-disabled):not(.ant-select-customize-input):not(.ant-pagination-size-changer):hover .ant-select-selector {
+  border-color: $color-main-lighter;
+}
+
+// INPUT FIELD FOR VALUE
+
+.ant-input:focus,
+.ant-input:focus-within {
+  border-color: $color-main-light;
+}
+
+.ant-input:hover {
+  border-color: $color-main-lighter;
+}
+
+.ant-picker:hover {
+  border-color: $color-main-lighter;
+}
+
+.ant-picker-focused.ant-picker {
+  border-color: $color-main-lighter;
+}
+
+// NUMBER TYPE INPUT FIELD
+
+.ant-input-number {
+  border-color: $color-main-light;
+}
+
+.ant-input-number:focus,
+.ant-input-number:focus-within {
+  border-color: $color-main-lighter;
+}
+
+// DATE-TIME PICKER
+
+.ant-picker-dropdown {
+  & a {
+    color: $color-main-light;
+  }
+
+  & .ant-btn-primary.ant-btn-disabled,
+  & .ant-btn-primary:disabled {
+    border: none;
+  }
+
+  & .ant-picker-cell-inner:hover {
+    background-color: $color-blue-light;
+  }
+
+  & .ant-picker-cell-today .ant-picker-cell-inner::before {
+    border-color: $color-main-light !important;
+  }
+
+  & .ant-picker-cell.ant-picker-cell-in-view.ant-picker-cell-selected {
+    & .ant-picker-cell-inner {
+      color: $color-main-dark;
+      background: $color-main-lighter;
+    }
+  }
+
+  & .ant-picker-time-panel-cell.ant-picker-time-panel-cell-selected {
+    & .ant-picker-time-panel-cell-inner {
+      background-color: $color-main-lighter !important;
+    }
+  }
+}
+
+.group {
+  background: $color-blue-light;
+  border: 1px solid $color-main-light;
+
+  &--conjunctions {
+    transform: translateX(1px);
+
+    & .ant-btn-group.ant-btn-group-sm {
+      background-color: $color-blue-lilac;
+      border-radius: 4px;
+    }
+
+    & .ant-btn.ant-btn-sm {
+      &:hover {
+        background-color: rgba(128, 128, 128, 0.1);
+      }
+    }
+
+    & .ant-btn-primary {
+      background-color: $color-main-light;
+
+      &:not(:disabled):not(.ant-btn-disabled):hover {
+        background-color: $color-main-lighter;
+      }
+    }
+
+    & .ant-btn-group>span:not(:first-child):not(:disabled),
+    & .ant-btn-group>.ant-btn-primary:not(:first-child):not(:disabled),
+    & .ant-btn-group>span:not(:first-child)>.ant-btn-primary:not(:disabled),
+    & .ant-btn-group>.ant-btn-primary:not(:first-child)>.ant-btn-primary:not(:disabled) {
+      border-inline-start-color: rgba(128, 128, 128, 0.1);
+    }
+
+    & .ant-btn-group>span:not(:last-child):not(:disabled),
+    & .ant-btn-group>.ant-btn-primary:not(:last-child):not(:disabled),
+    & .ant-btn-group>span:not(:last-child)>.ant-btn-primary:not(:disabled),
+    & .ant-btn-group>.ant-btn-primary:not(:last-child)>.ant-btn-primary:not(:disabled) {
+      border-inline-end-color: rgba(128, 128, 128, 0.1);
+    }
+  }
+}
+
+.group--actions.group--actions--tr {
+
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: end;
```

Comment:
let's use `flex-end`
I'm getting the following warning with `end`
`autoprefixer: end value has mixed support, consider using flex-end instead`

Reply:
d

Reply:
Chrome doesn't support 'end' yet, it looks like still Firefox and Opera are the only browsers to support it

Reply:
ok, so can we use `flex-end`?

Reply:
nope, it doesn't work in Chrome.